adoc/iv-Application/SA.adoc

```diff
@@ -19,7 +19,8 @@ Application workloads will consider the components, these will include, but not
 ////
 === SUSE Linux Enterprise Server for SAP Applications
 
-SUSE Linux Enterprise Server for SAP Applicatinos is a product formed from a bundle of software and services.  It is targeted specifically at customers running SAP workloads.  At it's foundation is SUSE Linux Enterprise Server and the High Availablity Extension with many additional components and benefits for running SAP Applications.
+SUSE Linux Enterprise Server for SAP Applications is a product formed from a bundle of software and services.  It is targeted specifically at customers running SAP workloads. At it's foundation is SUSE Linux Enterprise Server and the High Availability Extension with many additional components and benefits for running SAP Applications.
+// "At it's foundation" -> "The foundation is" ??
```

Comment:
@stephenmogg / @jamesongithub  - what is better?

Reply:
```suggestion
```
SUSE Linux Enterprise Server for SAP Applications (SLES for SAP) is a product formed from a bundle of software and services targeted specifically at customers running SAP workloads. A key component of SLES for SAP is the High Availability Extension which enables clustering technologies to eliminate singe points of failure in SAP applications.

Reply:
James, this does not match what i want to say. The base is SLES+HA...

Reply:
up to you.